src/osd/osd_types.cc

```diff
@@ -1482,12 +1484,17 @@ void pg_pool_t::encode(bufferlist& bl, uint64_t features) const
     return;
   }
 
-  uint8_t v = 24;
+  uint8_t v = 25;
   if (!(features & CEPH_FEATURE_NEW_OSDOP_ENCODING)) {
     // this was the first post-hammer thing we added; if it's missing, encode
     // like hammer.
     v = 21;
   }
+  if ((features &
+       (CEPH_FEATURE_RESEND_ON_SPLIT|CEPH_FEATURE_SERVER_KRAKEN)) !=
+      (CEPH_FEATURE_RESEND_ON_SPLIT|CEPH_FEATURE_SERVER_KRAKEN)) {
```

Comment:
oops, should have been JEWEL.  this is needed because we are recycling feature bits and older code will have these for other reasons.  we really need to introduce a set of helpers to manage this mess...